.github/workflows/upgrade-gradle-wrapper.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Gradle Wrapper Upgrade'
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 8 * * *'
+
+jobs:
+  upgrade-gradle-wrapper:
+    runs-on: ubuntu-latest
+    env:
+      WRAPPER_UPGRADE_GIT_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup git
+        run: |
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "github-actions[bot]"
+          git config --global url."https://unused-username:${WRAPPER_UPGRADE_GIT_TOKEN}@github.com/".insteadOf "https://github.com/"
```

Comment:
Sorry, one final thing, how is this supposed to work? Can you link me to the plugin docs? It seems to me that's exposing the token in the URL. Is that intended?

Reply:
Nvm, all clear

Reply:
Just for reference.
This the url to the plugin:
https://github.com/gradle/wrapper-upgrade-gradle-plugin

There is no official doc how to create the action. And honesty, I also don't know exactly why this has to be like this 😔. I guess this has something to do with how the plugin clones the project with got...

This is how it looks like in action:
![Screenshot_20230903-060441~2.png](https://github.com/Ivy-Apps/ivy-wallet/assets/10229883/23226923-032d-4324-ab7c-1741e778492a)